### DISCUSSION TITLE
ようこそ

### DISCUSSION THREAD

**Discussion by kenton116** [kenton116/n-development-team#2] [General]
## ようこそ！n-development-teamに！
自己紹介をして、早速チーム開発を始めましょう！

[自己紹介の例]
呼び名:kenton116
メッセージ:よろしくお願いします。

**Comment by ihwka** [comment]
name : ihwka(発音 イフカ)
message : これから宜しくお願いします。お手柔らかにw

**Reply by kenton116** [reply]
よろしくお願いします！

**Comment by fuuchin** [comment]
# よろしくお願いします！！  
自己紹介
------

- 呼び方： ふーちん
- 好きなもの：プログラミング、slack、果物
- 性格：やりたくないことをダラダラする
- プログラミングで何したい？：先生みたいになりたい！言語とか作ってみたい！

その他
-------

- 眼鏡をかけてます
- 一緒に活動できないこともありますが、時間があれば開発に力を入れたいと思います

**Reply by kenton116** [reply]
よろしくお願いします!

**Comment by moesuke** [comment]
### 自己紹介
なまえ→もえすけ
しゅみ→りょうり🍚
好きなポケモン→リザードン🔥

### メッセージ 
お昼間はお仕事をしていて、
元気が残っている日は夜にいろいろ勉強をしています。
マメに顔を出せるかはわかりませんが、どうぞよろしくおねがいいたします。

**Reply by kenton116** [reply]
よろしくお願いします！

**Comment by nrt0** [comment]
### 自己紹介
名前：てぃーでぃ（td）
趣味：自然観察

### メッセージ
休みの日に活動できたら良いなと思っています。
よろしくお願いします。

**Reply by kenton116** [reply]
よろしくお願いします！

**Comment by satsukizzz** [comment]
自己紹介！
名前：さつき
趣味：飲み物と音ゲーと読書
メッセージ
全体的に見たコードの支援とかできたら嬉しいです！

**Reply by kenton116** [reply]
よろしくお願いします!

**Comment by labeneko** [comment]
プログラミング講師の津野です。
こちらのチーム開発の頑張りを影で見守っております…！

困ったらSlackで相談してみてください！

**Reply by kenton116** [reply]
よろしくお願いします！

**Reply by kuriko** [reply]
同じく見守りという感じになると思いますがよろしくお願いします！ 🙏
